### ISSUE TITLE
TTS bible reading suggestion - scroll full page

### ISSUE THREAD

**Issue by luke-2136** [AndBible/and-bible#2069]
For TTS bible reading as it is now the TTS reads (and highlights) while staying at the top of the screen as the page clicks up verse by verse. For me at least, this makes it somewhat difficult to follow along. After each verse the whole page moves up and this creates a lot of continual movement. Could be a nice alternative to have the page stationary and have the TTS read down the page and highlight as it goes, then when it gets to the bottom of the screen go the next page and start again from the top.

There could be a number of advantages to users if this were at least an option. For instance the TTS could be paused at any point and what had just been read could be considered more carefully because it would still be on the screen. The way it works now though after each verse is read it is gone from the screen as the TTS jumps to the next verse. 

To be able to continue viewing the whole passage while the TTS is reading would also be useful for just following along if the user wanted to - and also would be much smoother.

Thanks again for And Bible

**Comment by agrogers** [comment]
Yes, I tend to agree that having it scroll on every verse was a little
annoying.

___________________________
*www.TenOutOfTen.org* <https://www.TenOutOfTen.org>
***@***.***
(+61) 0477 174 193 (Australia)
(+95) 09 250018669 (Myanmar)
